sections/product-main-template.liquid

```diff
@@ -285,18 +288,26 @@
     <li class="tab-options"><a href="#tab-3">Reviews</a></li>
     <li class="tab-options"><a href="#tab-4">Questions & Answers</a></li>
   </ul>
-  <hr style="margin-top:0px;">
+  <hr class="hr-no-margin">
   <div id="tab-1">
     {% include "product-features" %}
   </div>
   <div id="tab-2">
     {% include "product-specs" %}
   </div>
   <div id="tab-3">
+    <div class="tabs-margin-div">
+      <h4 style="color:black;">REVIEWS</h4>
+    <hr class="red-hr">
     {% include "bv-ratings-reviews" %}
+    </div>
   </div>
   <div id="tab-4">
+    <div class="tabs-margin-div">
+      <h4 style="color:black;">QUESTIONS & ANSWERS</h4>
```

Comment:
"Questions & Answers" should come from the locale file. I know this probably was a carryover from my initial BV implementation.